statsmodels/tsa/statespace/dynamic_regression.py

```diff
@@ -0,0 +1,321 @@
+"""
+Univariate time-varying coefficient regression model
+
+Author: Alastair Heggie
+License: Simplified-BSD
+"""
+import pdb
+import numpy as np
+from scipy.signal import lfilter
+from scipy.linalg import block_diag
+import statsmodels.api as sm
+import pandas as pd
+from .tools import (constrain_stationary_univariate,
+    unconstrain_stationary_univariate)
+import warnings
+
+# Construct the model
+class dynamic_regression(sm.tsa.statespace.MLEModel):
+    """
+    Univariate time-varying coefficient regression models
+    
+    A model in which the coefficients of the exogonous covariates are 
+    described by a univariate unobserved components model. At present
+    only local level and trend models are available for the coefficinets.
+    Ulitmately it is intended to add the full suite of components that
+    are standard in unobserved components models: 
+
+    Parameters
+    ----------
+    exog : array_like or None, optional
+        Exogenous variables.
+    exog_models : dict or list of dicts, optional
+        Either a dict, a list containing a single dict, or a list of dict
+        s, one for each exogonous regressor.
+        Admisble keys are the supported component types: 'irregular', 
+        'local_level', and 'trend'. If a key is included in the dict and 
```

Comment:
I think it should be `level` and `trend`, to match what `UnobservedComponents` has.